ethcore/src/evm/evm.rs

```diff
@@ -79,6 +82,12 @@ impl From<builtin::Error> for Error {
 	}	
 }
 
+impl From<wasm::RuntimeError> for Error {
+	fn from(_err: wasm::RuntimeError) -> Self {
+		Error::Wasm("Runtime error")
```

Comment:
I think i will do as you originally suggested for now, more proper way would require interpreter to be generic over error type that user functions can return